app/views/finders/_grouped_heading.mustache

```diff
@@ -0,0 +1 @@
+<h1 id="grouped-heading" class="govuk-heading-l search-results-title">Results</h1>
```

Comment:
Does this need to be in its own mustache template?

Reply:
As far as I can tell it gets used by `live_search.js` as the results count is rewritten. It's completely static though which kinda defeats the point, the other option is to put more exceptions in the live search javascript which seemed to be the worst of two bad options.
My 2p is we should choose 1 way of presenting the result count and do that for all finders. Perhaps some design review of this would be helpful.